packages/astro/test/types/call-action.ts

```diff
@@ -0,0 +1,51 @@
+import { describe, it } from 'node:test';
+import { expectTypeOf } from 'expect-type';
+import { type ActionReturnType, defineAction } from '../../dist/actions/runtime/virtual/server.js';
+import { z } from '../../zod.mjs';
+import type { APIContext } from '../../src/@types/astro.js';
```

Comment:
I suspect that your imports here are the reason why the type tests don't pass, importing from the project tend to result in the TypeScript project's scope adding things it shouldn't

Reply:
Ah ha, that's probably it. Thanks!